pkg/kube/client.go

```diff
@@ -152,7 +152,7 @@ func (c *Client) Get(namespace string, reader io.Reader) (string, error) {
 		return "", err
 	}
 	err = perform(c, namespace, infos, func(info *resource.Info) error {
-		log.Printf("Doing get for: '%s'", info.Name)
+		log.Printf("Doing get for %s: '%s'", info.Mapping.GroupVersionKind.Kind, info.Name)
```

Comment:
- [ ] Quoting the name with single quotation marks is odd.  
  Consider instead using the `%q` format verb and removing the single quotation marks.